CMakeLists.txt

```diff
@@ -182,7 +176,7 @@ add_subdirectory(third_parties/CLI11)
 #add_executable(meta-json-parser-benchmark benchmark/checkpoint_results.cpp benchmark/checkpoint_results.h)
 add_executable(meta-json-parser-benchmark)
 
-target_link_libraries(meta-json-parser-benchmark PRIVATE lib-meta-json-parser CLI11)
+target_link_libraries(meta-json-parser-benchmark PRIVATE lib-meta-json-parser fmt::fmt CLI11)
```

Comment:
otherwise throws linking errors

Reply:
During which part of build (I wonder which part requires `fmt` library)?